src/mscorlib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -2691,19 +2691,19 @@ public static IntPtr GetFunctionPointerForDelegate<TDelegate>(TDelegate d)
         [MethodImplAttribute(MethodImplOptions.InternalCall)]
         internal static extern IntPtr GetFunctionPointerForDelegateInternal(Delegate d);
 
-#if FEATURE_LEGACYSURFACE
-
-#if FEATURE_COMINTEROP
         [System.Security.SecurityCritical]  // auto-generated_required
         public static IntPtr SecureStringToBSTR(SecureString s) {
             if( s == null) {
                 throw new ArgumentNullException(nameof(s));
             }
             Contract.EndContractBlock();
             
-            return s.ToBSTR();
-        }
+#if FEATURE_COMINTEROP
+            return s.MarshalToBSTR();
+#else
+            return IntPtr.Zero;
```

Comment:
The problem is that the method is in netstandard2.0. 

We can either have it throw PNSE or remove it from the standard.

Or have 2 implementations (windows/unix).